src/modules/codeReview/CodeReviewBody.tsx

```diff
@@ -150,13 +150,35 @@ const CodeReviewBody: FC<Props> = () => {
                 <LineNoContainer>
                   {
                     // Control the visibility of lines' numbers, and "add comment" Button conditionally, by checking if we have empty content to review,
-                    // And if it's empty, then don't show anything 'as there is nothing to show to review on the screen',
+                    // And if it's empty, then just show "add comment" Button 'as there is nothing to show to review on the screen',
                     // And if it's not empty and the user has chosen to whether keep or hide lines' numbers, it will do that by whether including "<LineNo>" tag or not.
-                    line.every(
-                      (token) => token.content.trim().length === 0,
-                    ) ? null : (
+                    line.every((token) => token.content.trim().length === 0) ? (
+                      <div>
+                        {allowComments && (
```

Comment:
This means that:
- when the line is considered "empty" -> only the "add comment button" is shown (if it is allowed)
- otherwise: the line number is shown (if it is allowed) and the "Add comment button is shown (if it is allowed)

As we discussed, I think you can drop the check on the line content, as the user can choose to show line number and allow comments separately.